README.md

```diff
@@ -1,52 +1,167 @@
 # Ajax and Callbacks
-
 ## Objectives
-+ Explain what an API is and why it's used
-+ Explain what Ajax is and why it's used
-+ Make a `get` request using Ajax to append text to a page
-+ Explain what a callback is and why Ajax runs asynchronously
+- Explain what an API is and why it's used
+- Explain what Ajax is and why it's used
+- Make a get request using Ajax to append text to a page
+- Explain what a callback is and why Ajax runs asynchronously
+
+## Intro (half done)
+
+So far, you've been using a gem to access a web API.  Gems are responsible for handling all of the nitty-gritty.  They allow us to easily access/modify the data that's important to us. But what happens if the API we want to use doesn't have an associated gem?  We will need to handle everything ourselves.  If that's the case, we better learn more about what an API really is!
+
+## API Basics (done)
+
+We interact with web APIs through a set of urls. Each url defines a resource that we access.  The benefit of using a gem is that we don't have to build these urls ourselves.  The gems wrap the necessary code into neat little packages.  When we ask a gem for data, a gem will make a request to the API, handle the response and parse the data into a format we can use. Instead of relying on the magic of gems let's get our hands dirty.
```

Comment:
This paragraph is great — interspersing it with a couple of notes from the paragraph above would allow us to combine the two while making the relevance of APIs to the discussion of AJAX immediately clear.